### PR TITLE
Samples - Add explanation about pipeline failing intentionally

### PR DESCRIPTION
**Pull request by Ark-kun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3076)

### CODE REVIEW

File: backend/src/apiserver/config/sample_config.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   {
     "name": "[Tutorial] DSL - Control structures",
-    "description": "[source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures) Shows how to use conditional execution and exit handlers.",
+    "description": "[source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures) Shows how to use conditional execution and exit handlers. This pipeline will randomly fail to demonstrate that exit handler gets executed even in case of failure.",
```
Comment (joeliedtke) [inline backend/src/apiserver/config/sample_config.json]: Nit: 

that exit handler -> that the exit handler

**Comment by joeliedtke** [comment]
Thank you for adding this PR! This should help make the behavior of the pipeline clear for new users.

File: backend/src/apiserver/config/sample_config.json (hunk 1 above)
Comment (Ark-kun) [reply backend/src/apiserver/config/sample_config.json]: FIxed

**Pull request merged**